### ISSUE TITLE
Constant loop-back if using GTalk also.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#43]
```
What steps will reproduce the problem?
1. Sign into Gtalk on phone.
2. Communicate with gtalksms via external client.

What is the expected output? What do you see instead?

When communicating externally, messages sent and recieved are not duplicated on 
the phone unless the conversation is started by the other end and has yet to be 
answered.

However in this case ALL chat is relayed to gtalk on the phone.

What version of the product are you using? On what operating system?

v1.7

Please provide any additional information below.

Using notify to "main gmail" and notify with a separate jabber account.

If you attempt to mute the jabber account on the phone it will propagate the 
mute to all clients as a block.
```

Original issue reported on code.google.com by `lordst...@gmail.com` on 9 Jan 2011 at 4:11
- Blocking: #212

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately that's by design. I'm personally using 3 gmail accounts...
I've to see if we can use the resource string in xmpp connection to solve this 
problem.
```

Original comment by `Florent....@gmail.com` on 10 Jan 2011 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
Alright I resolved it by using a second jabber account, however as you could 
imagine it makes the system a little less awesome.

However all is good for now!
```

Original comment by `lordst...@gmail.com` on 11 Jan 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
An option to disable GoogleTalk when GTalkSMS is connected would also be a nice 
solution. Usually you don't need both running, not sure if it's doable.
```

Original comment by `miszobi@gmail.com` on 27 Jan 2011 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
Nothing we can do here. GTalk App provides no interface to control it's status. 
Personally I recommend not to use method 1, create an extra account for 
GTalkSMS instead.
```

Original comment by `fschm...@gmail.com` on 16 Apr 2011 at 2:45
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I think I found a way to prevent this. Not sure if the fix will hit GTalkSMS 
3.3 which is scheduled to be released this Friday.
```

Original comment by `fschm...@gmail.com` on 9 Nov 2011 at 9:53
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 9 Nov 2011 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in the current dev-version of GTalkSMS. We are going to 
release a new beta soon.
```

Original comment by `fschm...@gmail.com` on 16 Nov 2011 at 11:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
3.3.1 is available in the downloads section
```

Original comment by `fschm...@gmail.com` on 19 Nov 2011 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed and deployed with GTalkSMS 3.4
```

Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 3:20
- Changed state: **Deployed**
